common/frame.go

```diff
@@ -182,6 +203,8 @@ func (f *Frame) inflightRequestsLen() int {
 }
 
 func (f *Frame) clearLifecycle() {
+	f.log.Debugf("Frame:clearLifecycle", "tid:%s furl:%q", f.id, f.url)
```

Comment:
Am I wrong in correcting `tid`->`fid`? If so, what does `t` stand for?

I won't mention it again, but it's often used to log `Frame.id` in other places.

```suggestion
	f.log.Debugf("Frame:clearLifecycle", "fid:%s furl:%q", f.id, f.url)
```